### DISCUSSION TITLE
Twitter Clone Projesi

### DISCUSSION THREAD

**Discussion by bgunduz55** [acikkaynak/deprem-yardim-projesi#29] [Ideas]
Hem bant daraltma açısından hem de kullanıcıların aşina olduğu sosyal medya. Yeni projeler yapılsa dahi kullanıcılar, günlük hayatta aktif kullandıkları sosyal medyayı daha fazla tercih ediyor. FE ve Mobile flutter kısmını hazır satın alıp api entegrasyonları ve düzenlemeler yapabiliriz. Backend microservis mimarisi ile genişleyebilir şekilde yaparsak geriye sadece sunucu masrafı kalır. İlk etapta belirlenecek faz1 işlevlerini oluşturmak yeterli. Devamında mevcut projelerle data entegrasyonu vs. çalışılabilir.
Roller:
FE developer
BE developer
Mobile-Flutter developer
Tester
Devops
Analyse

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibin kontrolü sonrasında reddedildi.
